src/Microsoft.OData.Core/ODataLibraryCompatibility.cs

```diff
@@ -4,14 +4,15 @@
 // </copyright>
 //---------------------------------------------------------------------
 
+using System;
 using System.Diagnostics.CodeAnalysis;
 
 namespace Microsoft.OData
 {
     /// <summary>
     /// Library compatibility levels.
     /// </summary>
-    [SuppressMessage("Microsoft.Design", "CA1008:EnumsShouldHaveZeroValue")]
+    [Flags]
     public enum ODataLibraryCompatibility

```

Comment:
```suggestion
    public enum ODataLibraryCompatibility : long
```
To accommodate more flags...